2021-04-08 14:15:13,855 INFO    MainThread:364 [wandb_setup.py:_flush():69] setting env: {}
2021-04-08 14:15:13,855 INFO    MainThread:364 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-08 14:15:13,855 INFO    MainThread:364 [wandb_init.py:_log_setup():336] Logging user logs to /home/runner/Ndebele-language-model/nd_lang/wandb/run-20210408_141513-td9e5nea/logs/debug.log
2021-04-08 14:15:13,855 INFO    MainThread:364 [wandb_init.py:_log_setup():337] Logging internal logs to /home/runner/Ndebele-language-model/nd_lang/wandb/run-20210408_141513-td9e5nea/logs/debug-internal.log
2021-04-08 14:15:13,856 INFO    MainThread:364 [wandb_init.py:init():369] calling init triggers
2021-04-08 14:15:13,856 INFO    MainThread:364 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'input_shape': (13,), 'output_shape': 1, 'layer_size': 64, 'dropout_amount': 0.2, 'num_layers': 3}
2021-04-08 14:15:13,856 INFO    MainThread:364 [wandb_init.py:init():415] wandb.init() called when a run is still active
2021-04-08 14:15:13,975 INFO    MainThread:364 [wandb_run.py:finish():1167] finishing run zitha/nd_lang/it6texgx
2021-04-08 14:15:13,975 INFO    MainThread:364 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 0
2021-04-08 14:15:13,975 INFO    MainThread:364 [wandb_run.py:_restore():1466] restore
2021-04-08 14:15:16,039 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 2
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 71860
  total_bytes: 71860
}

2021-04-08 14:15:16,344 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 71860
  total_bytes: 75552
}

2021-04-08 14:15:16,447 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 75552
  total_bytes: 75552
}

2021-04-08 14:15:16,549 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 75552
  total_bytes: 75552
}

2021-04-08 14:15:16,651 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 75552
  total_bytes: 75552
}

2021-04-08 14:15:16,753 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  other_count: 2
}
pusher_stats {
  uploaded_bytes: 75552
  total_bytes: 75552
}

2021-04-08 14:15:18,037 INFO    MainThread:364 [wandb_run.py:_show_files():1838] logging synced files
2021-04-08 14:15:18,053 INFO    MainThread:364 [wandb_settings.py:_apply_source_login():724] setting login settings: {}
2021-04-08 14:15:18,061 INFO    MainThread:364 [wandb_setup.py:_flush():69] setting env: {}
2021-04-08 14:15:18,061 INFO    MainThread:364 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-08 14:15:18,061 INFO    MainThread:364 [wandb_init.py:_log_setup():336] Logging user logs to /home/runner/Ndebele-language-model/nd_lang/wandb/run-20210408_141518-1ck4an25/logs/debug.log
2021-04-08 14:15:18,061 INFO    MainThread:364 [wandb_init.py:_log_setup():337] Logging internal logs to /home/runner/Ndebele-language-model/nd_lang/wandb/run-20210408_141518-1ck4an25/logs/debug-internal.log
2021-04-08 14:15:18,062 INFO    MainThread:364 [wandb_init.py:init():369] calling init triggers
2021-04-08 14:15:18,062 INFO    MainThread:364 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'name': 'Blessing', 'surname': 'Chitakatira'}
2021-04-08 14:15:18,062 INFO    MainThread:364 [wandb_init.py:init():418] starting backend
2021-04-08 14:15:18,062 INFO    MainThread:364 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-08 14:15:18,065 INFO    MainThread:364 [backend.py:ensure_launched():123] starting backend process...
2021-04-08 14:15:18,070 INFO    MainThread:364 [backend.py:ensure_launched():127] started backend process with pid: 470
2021-04-08 14:15:18,071 INFO    MainThread:364 [wandb_init.py:init():423] backend started and connected
2021-04-08 14:15:18,074 INFO    MainThread:364 [wandb_init.py:init():463] updated telemetry
2021-04-08 14:15:18,074 INFO    MainThread:364 [wandb_init.py:init():482] communicating current version
2021-04-08 14:15:18,938 INFO    MainThread:364 [wandb_init.py:init():487] got version response 
2021-04-08 14:15:18,938 INFO    MainThread:364 [wandb_init.py:init():495] communicating run to backend with 30 second timeout
2021-04-08 14:15:19,020 INFO    MainThread:364 [wandb_init.py:init():520] starting run threads in backend
2021-04-08 14:15:20,631 INFO    MainThread:364 [wandb_run.py:_console_start():1524] atexit reg
2021-04-08 14:15:20,631 INFO    MainThread:364 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-08 14:15:20,631 INFO    MainThread:364 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-08 14:15:20,632 INFO    MainThread:364 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-08 14:15:20,633 INFO    MainThread:364 [wandb_init.py:init():544] run started, returning control to user process
2021-04-08 14:15:20,696 INFO    MainThread:364 [wandb_run.py:finish():1167] finishing run zitha/nd_lang/1ck4an25
2021-04-08 14:15:20,696 INFO    MainThread:364 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 0
2021-04-08 14:15:20,696 INFO    MainThread:364 [wandb_run.py:_restore():1466] restore
2021-04-08 14:15:22,783 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 14908
  total_bytes: 14908
}

2021-04-08 14:15:23,037 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 14908
  total_bytes: 18478
}

2021-04-08 14:15:23,140 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 18478
  total_bytes: 18478
}

2021-04-08 14:15:23,242 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 18478
  total_bytes: 18478
}

2021-04-08 14:15:23,344 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 18478
  total_bytes: 18478
}

2021-04-08 14:15:23,446 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 18478
  total_bytes: 18478
}

2021-04-08 14:15:24,725 INFO    MainThread:364 [wandb_run.py:_show_summary():1771] rendering summary
2021-04-08 14:15:24,725 INFO    MainThread:364 [wandb_run.py:_show_history():1809] rendering history
2021-04-08 14:15:24,726 INFO    MainThread:364 [wandb_run.py:_show_files():1838] logging synced files
2021-04-08 14:15:24,742 INFO    MainThread:364 [wandb_settings.py:_apply_source_login():724] setting login settings: {}
2021-04-08 14:15:24,751 INFO    MainThread:364 [wandb_setup.py:_flush():69] setting env: {}
2021-04-08 14:15:24,751 INFO    MainThread:364 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-08 14:15:24,751 INFO    MainThread:364 [wandb_init.py:_log_setup():336] Logging user logs to /home/runner/Ndebele-language-model/nd_lang/wandb/run-20210408_141524-kc9mp06c/logs/debug.log
2021-04-08 14:15:24,751 INFO    MainThread:364 [wandb_init.py:_log_setup():337] Logging internal logs to /home/runner/Ndebele-language-model/nd_lang/wandb/run-20210408_141524-kc9mp06c/logs/debug-internal.log
2021-04-08 14:15:24,751 INFO    MainThread:364 [wandb_init.py:init():369] calling init triggers
2021-04-08 14:15:24,751 INFO    MainThread:364 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-04-08 14:15:24,751 INFO    MainThread:364 [wandb_init.py:init():418] starting backend
2021-04-08 14:15:24,752 INFO    MainThread:364 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-08 14:15:24,755 INFO    MainThread:364 [backend.py:ensure_launched():123] starting backend process...
2021-04-08 14:15:24,759 INFO    MainThread:364 [backend.py:ensure_launched():127] started backend process with pid: 524
2021-04-08 14:15:24,761 INFO    MainThread:364 [wandb_init.py:init():423] backend started and connected
2021-04-08 14:15:24,764 INFO    MainThread:364 [wandb_init.py:init():463] updated telemetry
2021-04-08 14:15:24,764 INFO    MainThread:364 [wandb_init.py:init():482] communicating current version
2021-04-08 14:15:25,632 INFO    MainThread:364 [wandb_init.py:init():487] got version response 
2021-04-08 14:15:25,633 INFO    MainThread:364 [wandb_init.py:init():495] communicating run to backend with 30 second timeout
2021-04-08 14:15:25,710 INFO    MainThread:364 [wandb_init.py:init():520] starting run threads in backend
2021-04-08 14:15:27,458 INFO    MainThread:364 [wandb_run.py:_console_start():1524] atexit reg
2021-04-08 14:15:27,458 INFO    MainThread:364 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-08 14:15:27,458 INFO    MainThread:364 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-08 14:15:27,460 INFO    MainThread:364 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-08 14:15:27,460 INFO    MainThread:364 [wandb_init.py:init():544] run started, returning control to user process
2021-04-08 14:15:27,521 INFO    MainThread:364 [wandb_run.py:finish():1167] finishing run zitha/nd_lang/kc9mp06c
2021-04-08 14:15:27,521 INFO    MainThread:364 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 0
2021-04-08 14:15:27,521 INFO    MainThread:364 [wandb_run.py:_restore():1466] restore
2021-04-08 14:15:29,588 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 2
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 14908
  total_bytes: 14908
}

2021-04-08 14:15:29,731 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 14908
  total_bytes: 18317
}

2021-04-08 14:15:29,833 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 18317
  total_bytes: 18317
}

2021-04-08 14:15:29,935 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 18317
  total_bytes: 18317
}

2021-04-08 14:15:30,038 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 18317
  total_bytes: 18317
}

2021-04-08 14:15:30,141 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 18317
  total_bytes: 18317
}

2021-04-08 14:15:30,243 INFO    MainThread:364 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 18317
  total_bytes: 18317
}

2021-04-08 14:15:31,609 INFO    MainThread:364 [wandb_run.py:_show_files():1838] logging synced files
2021-04-08 14:15:31,612 INFO    MainThread:364 [wandb_run.py:finish():1167] finishing run zitha/nd_lang/it6texgx
